utils/text/transformers.py

```diff
@@ -168,16 +172,22 @@ def _repl(matchobj):
     return tokenized_title
 
 
-def snake_case_to_camel_case(string):
+def snake_case_to_camel_case(s):
     """Convert `snake_case` string to `CamelCase`"""
-    camel_string = ''.join(
-        part.capitalize() for part in string.lower().split('_')
-    )
+    camel_string = ''.join(part.capitalize() for part in s.lower().split('_'))
     return camel_string
 
 
-def snake_case_to_lower_camel_case(string):
+def snake_case_to_lower_camel_case(s):
     """Convert `snake_case` string to `camelCase`"""
-    camel_string = snake_case_to_camel_case(string)
+    camel_string = snake_case_to_camel_case(s)
     camel_string = camel_string[0].lower() + camel_string[1:]
-    return camel_string
\ No newline at end of file
+    return camel_string
+
+
+def pascal_case_to_snake_case(s):
```

Comment:
Does this also work for `camel_case_to_snake_case`? Anyway, let’s add a wrapper and make whichever one is the more basic one the base implementation that gets wrapped, and the more complex one the wrapper on the outer layer.

Reply:
No, it does not work unfortunately